extensions/sql-database-projects/src/models/options/deployOptionsModel.ts

```diff
@@ -23,7 +23,8 @@ export class DeployOptionsModel {
 	}
 
 	/**
-	 * Gets the options checkbox check value
+	 * Gets the options checkbox values by iterating through the labels and gets the default/changed value from the optionsMapTable
+	 * Returns data as [optionName, booleanValue], where these values are sending to the options table component
```

Comment:
the string[][] on the line below can be removed and replaced with this description